packages/react-core/src/components/Select/Select.tsx

```diff
@@ -53,6 +53,8 @@ export interface SelectProps extends MenuProps, OUIAProps {
   toggle: SelectToggleProps | ((toggleRef: React.RefObject<any>) => React.ReactNode);
   /** Flag indicating the toggle should be focused after a selection. If this use case is too restrictive, the optional toggleRef property with a node toggle may be used to control focus. */
   shouldFocusToggleOnSelect?: boolean;
+  /** Flag indicating the first menu item should be focused after opening the menu. */
+  shouldFocusFirstMenuItemOnOpen?: boolean;
```

Comment:
Just to shorten it a little:

```suggestion
  /** Flag indicating the first menu item should be focused after opening the menu. */
  shouldFocusFirstItemOnOpen?: boolean;
```